### PR TITLE
V1.9.10 - Updates

### PR DESCRIPTION
**Pull request by ClutchplateDude**
- Fixed a compiler bug when enabling LCD_BUTTON_TEST
- Allowed connection test to be set in local config

### CODE REVIEW

File: src/c_buttons.hpp [hunk 1]
```diff
@@ -21,7 +22,7 @@
   int lastLoopKey = -1;
 
   #if LCD_BUTTON_TEST == 1
```
Comment (julianneswinoga) [inline src/c_buttons.hpp:24]: We should add the button test to the matrix build to ensure this won't break in the future

File: Changelog.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
+**V1.9.10 - Updates**
+- Fixed a compiler bug when enabling LCD_BUTTON_TEST
```
Comment (julianneswinoga) [inline Changelog.md]: I would also say "compilation issue" rather than "compiler bug"... Thought you found an issue with GCC at first read :laughing:

Comment (ClutchplateDude) [reply Changelog.md]: LOL, totally.. must have been a lack of coffee...

File: src/c_buttons.hpp (hunk 1 above)
Comment (ClutchplateDude) [reply src/c_buttons.hpp:24]: Yeah, agreed. @andre-stefanov would this be easy to do?

**Review by CTetford — APPROVED** [review]

**Pull request merged**